community-plugins.json

```diff
@@ -499,10 +499,10 @@
     },
     {
         "id": "neo4j-graph-view",
-        "name": "Neo4j Graph View",
+        "name": "Juggl",
         "author": "Emile van Krieken",
-        "description": "A plugin for advanced graph visualization and querying using Neo4j.",
-        "repo": "HEmile/obsidian-neo4j-graph-view",
+        "description": "Adds a completely interactive, stylable and expandable graph view to Obsidian",
+        "repo": "HEmile/juggl",
```

Comment:
Is the plan to make the plugin into its own repo or leave it in the neo4j one? This destination current doesn't exist.

Reply:
I will rename the current repo to juggl when this PR is accepted.

Reply:
I cannot change it before it is accepted, otherwise the old plugin won't appear on the community plugins section anymore